operators/forklift-operator/2.6.4/metadata/annotations.yaml

```diff
@@ -0,0 +1,10 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: forklift-operator
+  operators.operatorframework.io.bundle.channels.v1: stable-v2.6
+  operators.operatorframework.io.bundle.channel.default.v1: stable-v2.6
+  operators.operatorframework.io.metrics.mediatype.v1: metrics+v1
+  com.redhat.openshift.versions: "v4.14-v4.15"
```

Comment:
also 4.16 should be included

Reply:
i'll try but that probably will be problematic. 4.16 doesn't exists in 2.6.3 (or any Z-stream before). while we use `replaces`. those two conflict.